src/protocols/secure_channel/tests/TestCASESession.cpp

```diff
@@ -67,6 +67,11 @@ P256SerializedKeypair accessoryOpKeysSerialized;
 
 P256Keypair commissionerOpKeys;
 P256Keypair accessoryOpKeys;
+
```

Comment:
Overall comment on TestCASESession.cpp:

Either:
- We need the DestinationId computation to be part of CHIPCryptoPAL.cpp/.h and test it in there
- We add a test case to TestCASESession.cpp to at least cover the spec test vector of destinationID computation.

Reply:
Added GenerateDestinationID test.